deployments/tls-app.yaml

```diff
@@ -24,17 +24,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
-            - name: POD_IP
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.podIP
           args:
-            - "-additional-dnsnames=example.com"
-            - "-cert-dir=/etc/tls"
             - "-namespace=$(NAMESPACE)"
-            - "-pod-ip=$(POD_IP)"
             - "-pod-name=$(POD_NAME)"
-            - "-service-names=tls-app"            
+            - "-query-k8s"
```

Comment:
Why control this based on a command line flag? Why not always query?

Reply:
I'm in contact with someone else who wants to use this project for client certificate use cases, where these semantics wouldn't be wanted.